tests/Console/ConfigurationResolverTest.php

```diff
@@ -961,6 +961,26 @@ public function provideResolveRulesCases()
                     '@A' => true,
                 ),
             ),
+            '@A - @C' => array(
```

Comment:
Not to me, I read it like this:
* `{ '@Foo': true, '@Bar': false }` => rules from `@Foo` that are not in `@Bar` => some rules from `@Foo`;
* `{ '@Bar': false }` => rules from nothing that are not in `@Bar` => nothing